config/openshift/operator.yaml

```diff
@@ -15,19 +15,39 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: tekton-operator
+  name: openshift-pipelines-operator
```

Comment:
Any reason for the name change here ? This is `tektoncd/operator` not the *downstream* openshift pipeline operator 👼🏼

Reply:
@nikhil-thomas I know it is 😝 but still, this is `tektoncd/operator`, not our *downstream* openshift-pipelines operator. This is the base of openshift-pipelines operator, but should it have the same name here, I am really not sure 🙃

Reply:
i shall work on this as part of configuring the operator bundle generation for openshift.
shall we merge this pr as it is. 🙂